pages/1.13/administering-clusters/deploying-a-local-dcos-universe/index.md

```diff
@@ -1,38 +1,38 @@
 ---
 layout: layout.pug
-navigationTitle:  Deploying a Local Universe
-title: Deploying a Local Universe
+navigationTitle:  Deploying a Local Catalog
+title: Deploying a Local Catalog
 menuWeight: 1000
-excerpt: Installing and running DC/OS services on a local Universe datacenter
+excerpt: Installing and running DC/OS services on a local Catalog datacenter
 enterprise: false
 ---
 
 <tr>
   <td align=justify style=color:black><strong>NOTE:</strong> If you are using dcos-enterprise, use <code>package-registry</code> for a seamless management of packages in air-gapped environments. Local universe support is deprecated.</td>
 </tr>
 
-You can install and run DC/OS services on a datacenter without internet access with a local [Universe](https://github.com/mesosphere/universe). You can deploy a local Universe that includes all Certified packages (basic), or a local Universe that includes selected packages (advanced).
+You can install and run DC/OS services on a datacenter without internet access with a local [Catalog](https://github.com/mesosphere/universe). You can deploy a local Catalog that includes all Certified packages (basic), or a local Catalog that includes selected packages (advanced).
```

Comment:
Should we ask engg to rename `universe` to `catalog` in https://github.com/mesosphere/universe?

Reply:
I'll ask Amr.